CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Reflection.Metadata.Ecma335;
+
+namespace CanHazFunny;
+
+public class Jester
+{
+    public IJokeService JokeService { get; }
+    public IJokeOutput JokeOutput { get; }
```

Comment:
Do these need to be publically accessible? Maybe make them private.
```suggestion
    private IJokeService _JokeService;
    private IJokeOutput JokeOutput;
```